src/docs/Components/Form/Toggle/code.mdx

```diff
@@ -71,9 +85,127 @@ Inside the label after you text :
 </div>
 ```
 
-Don't forget to add the `aria-hidden="true"` attribute on the `mc-toggle__state` element
+<Highlight theme="tips" title="Accessibility rule">
+
+  Don't forget to add the attribute `aria-hidden="true"` on the `mc-toggle__state` element
+
+</Highlight>
+
+<Preview path="toggle-states-label" />
+
+## Implement as a group
+
+### Import
+
+Import **the settings**, **the toggle** and **the fields** scss files.
+
+```scss
+@import 'settings-tools/_all-settings';
+
+@include import-font-families();
+
+@import 'components/_c.toggle';
+
+@import 'components/_c.fields';
+```
+
+> Note that the import order is important to get the right rendering of the component.
+
+### Group usage
+
+To use the toggle pattern in a group of elements in your form, you must use the following:
+
+- `mc-field` and the modifier `mc-field--group` on container (preferably use a fieldset tag)
+- `mc-field__legend` on the legend tag
+- `mc-field__container` on the wrapper of the set of toggle elements
+- and `mc-field__item` in addition of the `mc-toggle` class on the toggle wrapper:
+
+```html
+<div class="example">
+  <fieldset class="mc-field mc-field--group">
+    <legend class="mc-field__legend">
+      Group Label
+    </legend>
+
+    <div class="mc-field__container">
+      <div class="mc-toggle">
+        <input class="mc-toggle__input" id="toggle1" type="checkbox" />
+        <label class="mc-toggle__label" for="toggle1">
+          Label
+        </label>
+      </div>
+      <div class="mc-toggle">
+        <input class="mc-toggle__input" id="toggle2" type="checkbox" />
+        <label class="mc-toggle__label" for="toggle2">
+          Label
+        </label>
+      </div>
+      <div class="mc-toggle">
+        <input class="mc-toggle__input" id="toggle3" type="checkbox" />
+        <label class="mc-toggle__label" for="toggle3">
+          Label
+        </label>
+      </div>
+    </div>
+  </fieldset>
+</div>
+```
+
+<Preview path="group-default" />
+
+### Variations
+
+#### Horizontal group
+
+By default the toggles elements are positioned vertically in the group.
```

Comment:
By default toggles are vertically positioned in the group.

Reply:
fixed